src/ocean/util/app/ext/UnixSocketExt.d

```diff
@@ -109,10 +109,13 @@ public class UnixSocketExt : IApplicationExtension, IConfigExtExtension
     public void handle ( cstring command, cstring args,
                          void delegate (cstring) send_response )
     {
+        scope pred = (cstring v) { return v.length; };
```

Comment:
It's possible, I just found it uglier (same reason why we have `const NAME = "App"`).

I'm not sure I understand what you mean by:

> since it would make the existence of this clearer.